framework/src/bundle/BundleManifest.cpp

```diff
@@ -143,34 +177,41 @@ bool BundleManifest::Contains(const std::string& key) const
 Any BundleManifest::GetValue(const std::string& key) const
 {
   auto iter = m_Headers.find(key);
-  if (iter != m_Headers.end()) {
+  if (m_Headers.cend() != iter)
+  {
     return iter->second;
   }
   return Any();
 }
 
 Any BundleManifest::GetValueDeprecated(const std::string& key) const
 {
+  copy_deprecated_properties(m_Headers, m_PropertiesDeprecated);
   auto iter = m_PropertiesDeprecated.find(key);
-  if (iter != m_PropertiesDeprecated.end()) {
+  if (m_PropertiesDeprecated.cend() != iter)
+  {
     return iter->second;
   }
   return Any();
 }
 
 std::vector<std::string> BundleManifest::GetKeysDeprecated() const
 {
+  copy_deprecated_properties(m_Headers, m_PropertiesDeprecated);
   std::vector<std::string> keys;
-  for (AnyMap::const_iterator iter = m_PropertiesDeprecated.begin();
-       iter != m_PropertiesDeprecated.end();
-       ++iter) {
+  for (AnyMap::const_iterator iter = m_PropertiesDeprecated.cbegin();
+       iter != m_PropertiesDeprecated.cend();
+       ++iter)
+  {
```

Comment:
The formatting of the if statement does not follow convention.

Reply:
Done.